devito/ops/operator.py

```diff
@@ -24,6 +25,15 @@ def _specialize_iet(self, iet, **kwargs):
         ops_timing = Call(namespace['ops_timing_output'], [FunctionPointer("stdout")])
         ops_exit = Call(namespace['ops_exit'])
 
+        dims = []
+        for n, (section, trees) in enumerate(find_affine_trees(iet).items()):
+            dims.append(len(trees[n].dimensions))
```

Comment:
Does this dimensions take into account the time dimension? 

Because, if the variable has a time dimension it should not be used as we are only interested in space dimensions.

Reply:
i'd say it can't take a time dimension, unless there is a time dimension inside a time dimension, which i don't think could happen.

when we load an affine tree into `trees`, there is only spatial dimensions within `dimensions`